assemblies/assembly-enabling-authentication.adoc

```diff
@@ -1,38 +1,43 @@
-- Additional providers for accessing external resources.
-
-The {product} supports the following authentication providers:
-
-//Auth0:: `auth0`
-//Atlassian:: `atlassian`
-Microsoft Azure::
-`microsoft`
-//Microsoft Azure Easy Auth:: `azure-easyauth`
-//Bitbucket:: `bitbucket`
-//Bitbucket Server:: `bitbucketServer`
-//Cloudflare Access:: `cfaccess`
-GitHub:: `github`
-//GitLab:: `gitlab`
-//Google:: `google`
-//Google IAP:: `gcp-iap`
-Keycloak:: `oidc`
-//Okta:: `okta`
-//OAuth 2 Custom Proxy:: `oauth2Proxy`
-//OneLogin:: `onelogin`
-//SAML:: `saml`
-
-For each provider that you want to use, follow the dedicated procedure to complete the following tasks:
-
-. Set up the shared secret that the authentication provider and {product} require to communicate.
-. Configure {product} to use the authentication provider.
+Depending on your organization's security policies, you might require to identify and authorize users before giving them access to resources, such as {product}.
+
+In {product-short}, authentication and authorization are two separate processes:
+
+. Authentication defines the user identity, and passes on this information to {product-short}.
+Read following chapters to configure authentication in {product-short}.
+
+. Authorization defines what the authenticated identity can access or do in {product-short}.
+See link:{authorization-book-url}[{authorization-book-title}].
+
+To explore {product-short} features, you can skip configuring authentication and authorization. 
+After the default installation, you can log in as a Guest user and access all the features.
+
+The authentication system in {product-short} is handled by external authentication providers.
+
+{product-short} supports following authentication providers:
+
+* Red Hat Single-Sign On (RHSSO)
+* GitHub
+* Microsoft Azure
+
+To identify users in {product-short}, configure:
+
+* One (and only one) authentication provider for sign-in and identification.
+* Optionally, additional authentication providers for identification, to add more information to the user identity, or enable access to additional external resources.
+
+For each authentication provider, set up the shared secret that the authentication provider and {product-short} require to communicate, first in the authentication provider, then in {product-short}.
+
+{product-short} uses an internal user catalog to store user identity information.
+
+To explore the authentication system and use {product-short} without authorization policies, you can bypass the user catalog and start using {product-short} without provisioning the {product-short} user catalog.
+
+To get, store, and update additional user information, such as group or team ownership, with the intention to use this data to define authorization policies, provision users and groups in the {product-short} catalog.
+
+{product-short} uses a one-way synchronization system to provision users and groups from your authentication system to the {product-short} user catalog.
```

Comment:
@themr0c Could we place this under the "Important" tag?
```suggestion
[IMPORTANT]
====
{product-short} uses a one-way synchronization system to provision users and groups from your authentication system to the {product-short} user catalog.
====
```

Reply:
Implermented, but added the second sentence too.